docs/pages/eas-update/download-updates.mdx

```diff
@@ -0,0 +1,70 @@
+
+<Collapsible summary="What if I want to always block app startup until the latest update is downloaded?">
+
+We recommend against this strategy because the resulting user experince is extremely poor. Typically when a user is stuck waiting on a splash screen when booting an app, they will close the app and try again (and so downloading the update won't complete), or give up and use another app. When the users' device is connected to a slow network, even when there is no update they may have to wait several seconds or more to load the app. If ensuring that your users always have the latest version of your app is critical, you may want to explore one of the other strategies explained here.
+
+</Collapsible>
+
+<Collapsible summary="How can I disable the default behavior?">
+
+You can disable the default behavior by setting the [`checkAutomatically`](/versions/latest/sdk/updates/#updatescheckautomaticallyvalue) option to `NEVER` in the `updates` configuration. This will prevent the app from checking for updates and downloading them automatically.
+
+</Collapsible>
+
+## Checking for updates while the app is running
+
+You can use `Updates.checkForUpdateAsync()` to check for updates while the app is running. This will return a promise that resolves to a [`UpdateCheckResult` object](/versions/latest/sdk/updates/#updatecheckresult), with `isAvailable` set to `true` if an update is available, and information about the update in the [`manifest`](https://docs.expo.dev/versions/latest/sdk/manifests/#expoupdatesmanifest) property.
+
+If an update is available, you can use the `Updates.downloadAsync()` method to download the update. This will return a promise that resolves when the download is complete. Finally, you can use the `Updates.reloadAsync()` method to reload the app with the new version. The `useUpdates()` hook can also be used to monitor the state of the `expo-updates` package from a React component.
+
+<Collapsible summary="What are common patterns for checking for updates while the app is running?">
+
+- You can check for updates at various points in your app's lifecycle, such as [when it is foregrounded](https://reactnative.dev/docs/appstate) or at some interval. When an update is found, you may want to show a dialog to the user to prompt the user to update.
+- You can check for updates at launch and display your own custom loading screen, if it is very important for your use case to ensure that users always get the latest verison at launch.
+
+</Collapsible>
+
+## Checking for updates while the app is backgrounded
```

Comment:
maybe we could add a warning here about it being quite a new feature. should probably add a code snippet for how to do this too, since it's pretty straightforward (basically just https://github.com/chrfalch/expo-bg-update/blob/main/src/backgroundTask.ts without any of the logs)

Reply:
Agree - pushed an update to this PR (hope it is ok!)